src/mumble/AudioOutput.cpp

```diff
@@ -660,17 +660,30 @@ bool AudioOutput::mix(void *outbuff, unsigned int frameCount) {
 					}
 				}
 
+				bool noPosVolume =
+					(Global::get().s.fAudioMaxDistVolume == 0) && (len >= Global::get().s.fAudioMaxDistance);
+				float maxDistVolume = calcGain(1.0f, len);
+
 				for (unsigned int s = 0; s < nchan; ++s) {
 					const float dot = bSpeakerPositional[s]
 										  ? connectionVec.x * speaker[s * 3 + 0] + connectionVec.y * speaker[s * 3 + 1]
 												+ connectionVec.z * speaker[s * 3 + 2]
 										  : 1.0f;
-					// Volume on the ear opposite to the sound should never reach 0 in the real world.
-					// The gain is multiplied by 19/20 and 1/20 is added. This will have the effect
-					// of bringing the lowest value up to 1/20, while keeping the highest value at 1.
-					// E.g. calcGain() = 1; 1 * 19/20 + 1/20 = 0.95 + 0.05 = 1
-					// calcGain() = 0; 0 * 19/20 + 1/20 = 0 + 0.05 = 0.05
-					const float str     = svol[s] * (1 / 20.0 + (19 / 20.0) * calcGain(dot, len)) * volumeAdjustment;
+					float str;
+					if (!noPosVolume) {
+						// Volume on the ear opposite to the sound should never reach 0 in the real world.
+						// In the current contex, we know that sound reaches at least one ear. Therefore, we
+						// define the minimum volume as 1/4th of the theoretical maximum (ignoring the sound direction
+						// but taking distance into account) for _any_ ear.
+						str = svol[s] * std::min(1.0f, 0.25f * maxDistVolume + calcGain(dot, len)) * volumeAdjustment;
```

Comment:
```suggestion
						str = svol[s] * std::min(1.0f, 0.25f * maxDistVolume + 0.75f * calcGain(dot, len)) * volumeAdjustment;
```
Otherwise, we'll end up boosting the volume.
I think that in this case we don't even need the `min`, but it probably doesn't hurt anyway.

Even more elegantly (imo) would be something like this:
```suggestion
						constexpr float relMinVol = 1.0f / 4;
						str = svol[s] * std::min(1.0f, relMinVol * maxDistVolume + (1 - relMinVol) * calcGain(dot, len)) * volumeAdjustment;
```
as this only requires a single number hard-coded in the source code instead of two (0.25 and 0.75)